### DISCUSSION TITLE
Exercise 6

### DISCUSSION THREAD

**Discussion by Irenekyei1** [LiNk-NY/IntroR#42] [General]
Hi @ralythompson24, do you recommend I use google colab for exercise 6 or RStudio instead? I'm asking because I am having difficulty uploading the .ipynb file to colab.

![image](https://github.com/CUNYSPHCode/IntroR/assets/171519954/8b715776-0eb0-4ef6-bbdc-66b5f49653f7)

**Comment by ralythompson24** [comment]
Hi @Irenekyei1 - you should use Google Colab since RStudio can't open .ipynb files.

It looks like a network error which means that either Google Colab is down or your internet connection is unstable. Maybe wait a few hours and try it again. Let me know if you still have issues after that.
